enterprise/micronaut/src/org/netbeans/modules/micronaut/MicronautConfigProperties.java

```diff
@@ -60,6 +94,16 @@ public static boolean hasConfigMetadata(Project project) {
                     Exceptions.printStackTrace(ex);
                 }
             }
+            String customRepository = getCustomRepository(project);
+            if (customRepository != null) {
+                try {
+                    InputStream stream = new ByteArrayInputStream(customRepository.getBytes(StandardCharsets.UTF_8));
+                    ConfigurationMetadataRepository repository = ConfigurationMetadataRepositoryJsonBuilder.create().withJsonResource(stream).build();
+                    props.putAll(repository.getAllProperties());
+                } catch (Exception ex) {
+                    Exceptions.printStackTrace(ex);
```

Comment:
shouldn't exception's propagete instead of being swallowed?

Reply:
The same as above.

Reply:
The same as above.